examples/UnreleasedAI/UnreleasedAI.md

```diff
@@ -0,0 +1,163 @@
+
+Creating effective prompts can be challenging. In Braintrust, you can edit your prompt directly in the UI and immediately see the changes in your application. For example, edit the **Generate changelog** prompt to include a friendly message at the end of the changelog:
+> Summarize the following commits from `{{url}}` since `{{since}}` in changelog form. Include a summary of changes at the top since the provided date, followed by individual pull requests (be concise). At the end of the changelog, include a friendly message to the user.
+>
+>`{{commits}}`
+
+Save the prompt, and it will be automatically updated in your your app – try it out! If you’re curious, you can also change the model here. The ability to iterate on and test your prompt is great, but writing a prompt in Braintrust is more powerful than that. Every prompt you create in Braintrust is also an AI function that you can invoke inside of your application.
+
+## Running a prompt as a function
+
+Running a prompt as an AI function is a faster and simpler way to iterate on your prompts and decide which model is right for your use case, and it comes out-of-the-box in Braintrust. Normally, you would need to choose a model upfront, hardcode the prompt text, and manage boilerplate code from various SDKs and observability tools. Once you create a prompt in Braintrust, you can invoke it with the arguments you created.
+
+In [app/generate/route.ts](http://route.ts), the prompt is invoked with three arguments: `url`, `since`, and `commits`.
+
+```typescript
+return await invoke({  
+    projectName: PROJECT_NAME,  
+    slug: PROMPT_SLUG,  
+    input: {  
+        url,  
+        since,  
+        commits: commits.map(({ commit }) => \`${commit.message}\\n\\n\`),  
+    },  
+    stream: true,  
+    });  
+});
+```
+
+To set up streaming and make sure the results are easy to parse, just set `stream` to `true`. The result of the function call is then shown to the user in the frontend of the application.
+
+Running a prompt as an AI function is also a powerful way to automatically set up other Braintrust capabilities. Behind the scenes, Braintrust automatically caches and optimizes the prompt through the [AI proxy](https://www.braintrust.dev/docs/guides/proxy) and logs it to your project, so you can dig into the responses and understand if you need to make any changes. This also makes it easy to change the model in the Braintrust UI, and automatically deploy it to any environment which invokes it.
+
+## Observability
+
+Traditional observability tools monitor performance and pipeline issues, but generative AI projects require deeper insights to ensure your application works as intended. As you continue using the application to generate changelogs for various GitHub repositories, you’ll notice every function call is [logged](https://www.braintrust.dev/docs/guides/logging), so you can examine the input and output of each call. 
+
+![Logs](assets/logs.png)
```

Comment:
Not critical, but we generally make the screenshots in our docs light mode (so they look particularly good on dark mode docs).

If you do change that, I think it'd be useful to make this a GIF or click into one of the logs and show the expanded version. It's nice to see that you can see a richer version of the inputs & outputs.